Stay aware/Sources/Scenes/Controller/RecordsViewController.swift

```diff
@@ -0,0 +1,335 @@
+        df.dateFormat = "HH:mm"
+        return df
+    }()
+    
+    lazy var monthFormatter: DateFormatter = {
+        let df = DateFormatter()
+        df.dateFormat = "MM"
+        return df
+    }()
+}
+
+//MARK: - UITableView
+extension RecordsViewController: UITableViewDelegate, UITableViewDataSource {
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 30
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 90 // Возвращает высоту 80 пунктов для всех ячеек
+    }
+    
+    func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
+        guard let headerView = view as? HeaderView else { return }
+        
+        // Установите нужные свойства для проматывания заголовка секции
+        headerView.layer.zPosition = 0 // Гарантирует, что заголовок будет проматываться
+        headerView.contentView.backgroundColor = UIColor(red: 234/255, green: 234/255, blue: 240/255, alpha: 1.0) // Настройте цвет фона, если требуется
+    }
+    
+    
+    func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        let view = tableView.dequeueReusableHeaderFooterView(withIdentifier: HeaderView.identifier) as! HeaderView
+        view.label.text = "\(days[section].first?.day ?? "")"
+        
+        return view
+    }
+    
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return days.count
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return days[section].count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "recordTableViewCell", for: indexPath) as? RecordTableViewCell
+        
+        // избавляемся от белого фона
+        cell?.backgroundColor = .clear
+        cell?.contentView.backgroundColor = .clear
+        cell?.backgroundView = nil // Удаление фонового представления
+        cell?.selectedBackgroundView = nil // Удаление фонового представления для выбранной ячейки
+        cell?.selectionStyle = .none // выкл выделение при нажатиии
+        
+        cell?.model = days[indexPath.section][indexPath.row]
+        return cell ?? UITableViewCell()
+    }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        
+        let detailVC = DetailViewController()
+        detailVC.delegate = self
+        
+        let objectID = days[indexPath.section][indexPath.row].id
+        
+        detailVC.context = context
+        detailVC.objectID = objectID
+        self.present(detailVC, animated: true)
+    }
+}
+
+//MARK: - UICollectionView
+extension RecordsViewController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
```

Comment:
```suggestion
// MARK: - UICollectionViewDelegate
extension RecordsViewController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
```
вынести в разные экстенжены
Делается для более простого нахождения требуемого куска кода